devices/generic/items/config_color_gradient_segments_item.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": "resourceitem1.schema.json",
+  "id": "config/color/gradient/segments",
+  "datatype": "UInt8",
+  "access": "RW",
+  "public": true,
+  "description": "Number of 20cm segments on Aqara LED Strip T1."
```

Comment:
The 20cm is reported through `cap/color/gradient/pixel_length` and shouldn't be part of the description.  Also, this would lead to a new item for each new device.